src/Nethermind/Nethermind.HealthChecks/HealthChecksPlugin.cs

```diff
@@ -63,7 +63,7 @@ public void AddServices(IServiceCollection service)
                     args: new object[] { _nodeHealthService });
             if (_healthChecksConfig.UIEnabled)
             {
-                service.AddHealthChecksUI(setup =>
+                service.AddHealthChecksUI(setupSettings: setup =>
```

Comment:
This change was necessary because of the upgrade from `AspNetCore.HealthChecks.UI` 3.x to 5.x.  I am not actually familiar with this syntax so I don't know what it is doing, but whatever it is doing it seemed to be necessary for things to compile.  😬